### PR TITLE
week 03-day-03 mod-01

### PR DESCRIPTION
**Pull request by ben-manning**
I have the base lecture down.  Now I'm just making sure the syntax is all good and that it matches Cameron's afternoon lecture.

### CODE REVIEW

**Comment by ben-manning** [comment]
Everything has been edited down and should match up nicely with Cameron's afternoon lecture.

File: week-03-resetful-server-express/day-03/module-01/readme.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+* <a href="http://mochajs.org" target="_blank">Mocha</a> - framework for running tests
+* <a href="http://chaijs.com/api" target="_blank">Chai</a> **(expect)** - for *expect* assertions
+* <a href="https://github.com/request/request" target="_blank">Request</a> - for handling HTTP requests and responses
+
+## Challenges
+
+1. Begin working on your <a href="https://github.com/sf-wdi-24/express-todo-app" target="_blank">express-todo-app</a>.
+2. When you're ready, use the steps above to implement routes for creating and reading todos.
+3. Make sure your routes work by using Postman.
+4. Write request specs to test your update and delete routes.
```
Comment (cameronjacoby) [inline week-03-resetful-server-express/day-03/module-01/readme.md]: Change to create and read

File: week-03-resetful-server-express/day-03/module-01/readme.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+## Resources
+
+* <a href="http://expressjs.com/api.html#req" target="_blank">Express Request Docs</a>
+* <a href="http://expressjs.com/api.html#app.put.method" target="_blank">Express app.put()</a>
```
Comment (cameronjacoby) [inline week-03-resetful-server-express/day-03/module-01/readme.md]: Change to `app.get()` and `app.post()` resources

**Comment by cameronjacoby** [comment]
Add `solutions.md` that links to `solution` branch of `express-todo-app` repo; also add link to week 3 schedule

**Pull request merged**